### ISSUE TITLE
Some bank tiles non-negotiable

### ISSUE THREAD

**Issue by ZephORSC** [Open-RSC/Game#1361]
**Describe the bug**
A few bank tiles are not able to be walked on. Have seen in the past the tile in Varrock west bank not be able to be walked on by myself, but able to by another player. Probably related to the bank counters they are next to.

**To Reproduce**
1. Go to locations
2. Attempt to walk on or through tile.

**Expected behavior**
Standard pathing through and standing on the tile.

**Screenshots**

Edgeville1:
https://gyazo.com/96d2a5960149b4b785bbf9f48b04e6f6

Edgeville2: 
https://gyazo.com/a281e3764569e38ed0f5ec7739169cb9

Varrock West1:
https://gyazo.com/3df80a075380c41ea3ca3037a7048656

Varrock West2:
https://gyazo.com/483bbefc685f704bfa9f2667d8937a9f

**Additional comments**
None found in Fally banks, Al Kharid, Ardy banks, Catherby, Seers, Fishing Guild, and Yanille.

There's these post tiles in Ardy north and Yanille, but I could believe that to be kosher:
https://gyazo.com/2ef2dc81560d4d21986b1c62297eeb97

**Comment by Marwolf** [comment]
I noticed this activity earlier but thought it was NPC blocking the path. Must be pathing around objects like the bank desks that is causing this.

**Comment by Christofosho** [comment]
I investigated this. It has to do with the rotation of the object not being accounted for properly by the client. This probably means that the server is not sending the information to the client. That is as far as I got. The next steps:
1) Check to see if the rotation information is sent to the client, and when.
2) Check to see if the client uses that rotation information properly (also called "direction" or "dir").
